2025-06-30 13:54:12,214 - INFO - User 923869729 (grndsard) started the bot
2025-06-30 13:54:13,102 - INFO - Welcome message sent to user 923869729 in en language
2025-06-30 13:55:33,122 - INFO - User 923869729 (grndsard) clicked: 5_11
2025-06-30 13:55:42,873 - INFO - User 923869729 (grndsard) clicked: yes_more
2025-06-30 13:56:01,635 - INFO - User 923869729 (grndsard) clicked: 2_12
2025-06-30 13:56:07,697 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-06-30 15:04:05,148 - INFO - User 1391675035 (Unknown) started the bot
2025-06-30 15:04:06,025 - INFO - Welcome message sent to user 1391675035 in en language
2025-06-30 15:04:14,275 - INFO - User 1391675035 (Unknown) clicked: 2_1
2025-06-30 15:08:33,367 - INFO - User 923869729 (grndsard) started the bot
2025-06-30 15:08:33,840 - INFO - Welcome message sent to user 923869729 in en language
2025-06-30 15:09:04,875 - INFO - User 923869729 (grndsard) clicked: 7_2
2025-06-30 15:09:13,129 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-06-30 15:44:38,916 - INFO - User 923869729 (grndsard) started the bot
2025-06-30 15:44:39,757 - INFO - Welcome message sent to user 923869729 in en language
2025-06-30 15:45:00,889 - INFO - User 923869729 (grndsard) clicked: 3_3
2025-06-30 15:45:10,452 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-04 11:10:17,764 - CRITICAL - Bot crashed: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
Traceback (most recent call last):
  File "D:\Datenordner\Datenprogrammierung\rupaganti\rupaganti_bot.py", line 382, in <module>
    bot.polling(none_stop=True)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1040, in polling
    logger.info('Starting your bot with username: [@%s]', self.user.username)
                                                          ^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 273, in user
    self._user = self.get_me()
                 ^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1236, in get_me
    result = apihelper.get_me(self.token)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 196, in get_me
    return _make_request(token, method_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
2025-07-04 15:16:35,344 - CRITICAL - Bot crashed: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
Traceback (most recent call last):
  File "D:\Datenordner\Datenprogrammierung\rupaganti\rupaganti_bot.py", line 382, in <module>
    bot.polling(none_stop=True)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1040, in polling
    logger.info('Starting your bot with username: [@%s]', self.user.username)
                                                          ^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 273, in user
    self._user = self.get_me()
                 ^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1236, in get_me
    result = apihelper.get_me(self.token)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 196, in get_me
    return _make_request(token, method_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
2025-07-09 14:29:58,045 - CRITICAL - Bot crashed: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
Traceback (most recent call last):
  File "D:\Datenordner\Datenprogrammierung\rupaganti\rupaganti_bot.py", line 382, in <module>
    bot.polling(none_stop=True)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1040, in polling
    logger.info('Starting your bot with username: [@%s]', self.user.username)
                                                          ^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 273, in user
    self._user = self.get_me()
                 ^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1236, in get_me
    result = apihelper.get_me(self.token)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 196, in get_me
    return _make_request(token, method_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
2025-07-24 09:11:44,477 - CRITICAL - Bot crashed: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
Traceback (most recent call last):
  File "D:\Datenordner\Datenprogrammierung\rupaganti\rupaganti_bot.py", line 514, in <module>
    bot.polling(none_stop=True)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1040, in polling
    logger.info('Starting your bot with username: [@%s]', self.user.username)
                                                          ^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 273, in user
    self._user = self.get_me()
                 ^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1236, in get_me
    result = apihelper.get_me(self.token)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 196, in get_me
    return _make_request(token, method_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
2025-07-24 09:14:35,446 - INFO - RupaGanti Bot starting...
2025-07-24 09:14:36,070 - CRITICAL - Bot crashed: A request to the Telegram API was unsuccessful. Error code: 404. Description: Not Found
Traceback (most recent call last):
  File "D:\Datenordner\Datenprogrammierung\rupaganti\rupaganti_bot.py", line 515, in <module>
    bot.polling(none_stop=True)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1040, in polling
    logger.info('Starting your bot with username: [@%s]', self.user.username)
                                                          ^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 273, in user
    self._user = self.get_me()
                 ^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1236, in get_me
    result = apihelper.get_me(self.token)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 196, in get_me
    return _make_request(token, method_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 404. Description: Not Found
2025-07-24 09:15:47,643 - INFO - RupaGanti Bot starting...
2025-07-24 09:15:48,196 - CRITICAL - Bot crashed: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
Traceback (most recent call last):
  File "D:\Datenordner\Datenprogrammierung\rupaganti\rupaganti_bot.py", line 529, in <module>
    bot.polling(none_stop=True, interval=1, timeout=20)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1040, in polling
    logger.info('Starting your bot with username: [@%s]', self.user.username)
                                                          ^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 273, in user
    self._user = self.get_me()
                 ^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1236, in get_me
    result = apihelper.get_me(self.token)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 196, in get_me
    return _make_request(token, method_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 401. Description: Unauthorized
2025-07-24 09:16:21,029 - INFO - RupaGanti Bot starting...
2025-07-24 09:16:26,803 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-24 09:16:26,804 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 1073, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Python311\Lib\site-packages\telebot\util.py", line 108, in raise_exceptions
    raise self.exception_info
  File "C:\Python311\Lib\site-packages\telebot\util.py", line 90, in run
    task(*args, **kwargs)
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 649, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\__init__.py", line 623, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 321, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-07-24 09:16:32,500 - INFO - RupaGanti Bot starting...
2025-07-24 09:16:51,367 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 09:16:52,452 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 09:17:26,210 - INFO - User 923869729 (grndsard) clicked: 8_1
2025-07-24 09:17:27,938 - ERROR - PDF to Word conversion error: [WinError 32] The process cannot access the file because it is being used by another process: 'temp/temp_1.pdf'
2025-07-24 09:17:35,191 - INFO - User 923869729 (grndsard) clicked: yes_more
2025-07-24 09:18:48,685 - INFO - User 923869729 (grndsard) clicked: 4_2
2025-07-24 09:18:53,033 - INFO - User 923869729 (grndsard) clicked: 2_3
2025-07-24 09:19:03,432 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-24 09:36:52,334 - INFO - Secure RupaGanti Bot starting with enhanced encryption...
2025-07-24 09:36:52,335 - INFO - Deleted temp file: temp\converted_1.docx
2025-07-24 09:36:52,335 - ERROR - Could not delete file temp\converted_1.docx: [WinError 2] The system cannot find the file specified: 'temp\\converted_1.docx'
2025-07-24 09:36:52,336 - INFO - Deleted temp file: temp\temp_1.pdf
2025-07-24 09:36:52,336 - ERROR - Could not delete file temp\temp_1.pdf: [WinError 2] The system cannot find the file specified: 'temp\\temp_1.pdf'
2025-07-24 09:36:52,336 - INFO - Initial cleanup completed - all previous files deleted
2025-07-24 09:37:09,807 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 09:37:10,980 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 09:37:59,018 - INFO - File securely stored: Nobis_043_NB_GM_KPU_HCM_1_III_2025_01_Perihal_Undangan_Pelatihan.pdf → 30bdf87b-243f-43d2-b84f-4bc6bb97eaa2.pdf
2025-07-24 09:38:01,829 - INFO - User 923869729 (grndsard) clicked: 8_4
2025-07-24 09:38:04,075 - WARNING - Could not delete temp PDF: [WinError 32] The process cannot access the file because it is being used by another process: 'temp/temp_4_1753324682.pdf'
2025-07-24 09:38:04,077 - INFO - Original file deleted after processing: files/30bdf87b-243f-43d2-b84f-4bc6bb97eaa2.pdf
2025-07-24 09:38:40,723 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-24 09:43:26,483 - INFO - Secure RupaGanti Bot starting with enhanced encryption...
2025-07-24 09:43:26,489 - INFO - Deleted temp file: temp\temp_4_1753324682.pdf
2025-07-24 09:43:26,492 - INFO - Database cleaned on startup
2025-07-24 09:43:26,492 - INFO - Initial cleanup completed - all previous files deleted
2025-07-24 09:43:43,110 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 09:43:44,268 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 09:44:10,697 - INFO - File securely stored: Justi-Kebutuhan_Pembelian_Template_Hyper_signed.pdf → c4da0452-d4e2-48ff-a316-52eb3187ae1a.pdf
2025-07-24 09:44:13,453 - INFO - User 923869729 (grndsard) clicked: 8_1
2025-07-24 09:44:15,095 - ERROR - PDF to Word conversion error: [WinError 267] The directory name is invalid: 'C:\\Users\\Grandis\\AppData\\Local\\Temp\\tmpw10eahj3\\temp_1.pdf'
2025-07-24 09:50:50,540 - INFO - Secure RupaGanti Bot starting with enhanced encryption...
2025-07-24 09:50:50,544 - INFO - Database cleaned on startup
2025-07-24 09:50:50,545 - INFO - Startup cleanup: Deleted files\c4da0452-d4e2-48ff-a316-52eb3187ae1a.pdf
2025-07-24 09:50:50,545 - INFO - Initial cleanup completed - all previous files deleted
2025-07-24 09:50:54,746 - INFO - User 923869729 (grndsard) clicked: yes_more
2025-07-24 09:51:08,422 - INFO - File securely stored: FREQUENTLY ASKED QUESTIONS (FAQ) DISKUSI UMUM.pdf → f87f47d3-ef52-4c69-a49a-6c7934df005a.pdf
2025-07-24 09:51:18,487 - INFO - User 923869729 (grndsard) clicked: 8_1
2025-07-24 09:51:18,505 - ERROR - PDF to Word conversion error: cannot access local variable 'PDF_TO_DOCX_AVAILABLE' where it is not associated with a value
2025-07-24 09:54:50,083 - INFO - Secure RupaGanti Bot starting with enhanced encryption...
2025-07-24 09:54:50,097 - INFO - Database cleaned on startup
2025-07-24 09:54:50,100 - INFO - Startup cleanup: Deleted files\f87f47d3-ef52-4c69-a49a-6c7934df005a.pdf
2025-07-24 09:54:50,100 - INFO - Initial cleanup completed - all previous files deleted
2025-07-24 09:54:53,056 - INFO - User 923869729 (grndsard) clicked: yes_more
2025-07-24 09:55:36,410 - INFO - File securely stored: an ATS CV - Template.docx.pdf → 85112017-43e3-4724-8a60-27ce31ae7dc3.pdf
2025-07-24 09:55:39,290 - INFO - User 923869729 (grndsard) clicked: 8_1
2025-07-24 09:55:41,685 - ERROR - Failed to clean up temp directory: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\Grandis\\AppData\\Local\\Temp\\rupaganti_nfcr_jbo\\temp_1.pdf'
2025-07-24 09:55:42,410 - INFO - Original file deleted after processing: files/85112017-43e3-4724-8a60-27ce31ae7dc3.pdf
2025-07-24 10:54:23,851 - INFO - Secure RupaGanti Bot starting with enhanced encryption...
2025-07-24 10:54:23,864 - INFO - Database cleaned on startup
2025-07-24 10:54:23,865 - INFO - Initial cleanup completed - all previous files deleted
2025-07-24 11:05:56,415 - INFO - Secure RupaGanti Bot starting with enhanced encryption...
2025-07-24 11:05:56,418 - INFO - Database cleaned on startup
2025-07-24 11:05:56,419 - INFO - Initial cleanup completed - all previous files deleted
2025-07-24 11:06:31,904 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 11:06:34,762 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 11:07:40,571 - INFO - File securely stored: Update Pricelist Gadget Terbaru Kisel.pdf → e533db8b-7e8d-4845-9a3a-6fbb79dedfc4.pdf
2025-07-24 11:09:42,856 - INFO - Deleted expired file: files/e533db8b-7e8d-4845-9a3a-6fbb79dedfc4.pdf
2025-07-24 11:17:13,289 - INFO - User 923869729 (grndsard) clicked: start_over
2025-07-24 11:18:14,550 - INFO - File securely stored: Update Pricelist Gadget Terbaru Kisel.pdf → c1e46415-eb5d-4544-9182-0f3d2fa81862.pdf
2025-07-24 11:18:24,638 - INFO - User 923869729 (grndsard) clicked: 5_1
2025-07-24 11:18:27,787 - INFO - Original file deleted after processing: files/c1e46415-eb5d-4544-9182-0f3d2fa81862.pdf
2025-07-24 11:24:51,207 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-24 11:24:55,932 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 11:24:56,785 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 11:25:56,166 - INFO - File securely stored: Update Pricelist Gadget Terbaru Kisel.pdf → 80f7ab27-5d64-4b8b-bd22-5b324087d475.pdf
2025-07-24 11:26:02,942 - INFO - User 923869729 (grndsard) clicked: 5_2
2025-07-24 11:26:06,199 - INFO - Original file deleted after processing: files/80f7ab27-5d64-4b8b-bd22-5b324087d475.pdf
2025-07-24 11:26:19,313 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-24 11:41:00,483 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 11:41:03,806 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 11:41:31,970 - INFO - File securely stored: Nobis_043_NB_GM_KPU_HCM_1_III_2025_01_Perihal_Undangan_Pelatihan.pdf → faf522f7-b497-4f52-8641-6b6a6c0af8ff.pdf
2025-07-24 11:43:36,588 - INFO - Deleted expired file: files/faf522f7-b497-4f52-8641-6b6a6c0af8ff.pdf
2025-07-24 11:44:20,115 - INFO - User 923869729 (grndsard) clicked: start_over
2025-07-24 11:55:21,188 - INFO - User 923869729 (grndsard) started the bot
2025-07-24 11:55:24,495 - INFO - Welcome message sent to user 923869729 in en language
2025-07-24 11:55:59,721 - INFO - File securely stored: 20250220092521394.pdf → 231b0bf7-39ce-4505-b2e0-628aa104a884.pdf
2025-07-24 11:56:08,060 - INFO - User 923869729 (grndsard) clicked: 5_1
2025-07-24 11:56:10,077 - INFO - Original file deleted after processing: files/231b0bf7-39ce-4505-b2e0-628aa104a884.pdf
2025-07-24 11:58:44,065 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-25 08:28:25,497 - INFO - Secure RupaGanti Bot starting with enhanced AES-256 with hardware acceleration encryption...
2025-07-25 08:28:25,526 - INFO - Database cleaned on startup
2025-07-25 08:28:25,527 - INFO - Initial cleanup completed - all previous files deleted
2025-07-25 08:28:25,527 - CRITICAL - Invalid bot token format. Please check your token.
2025-07-25 08:29:11,894 - INFO - Secure RupaGanti Bot starting with enhanced AES-256 with hardware acceleration encryption...
2025-07-25 08:29:11,901 - INFO - Database cleaned on startup
2025-07-25 08:29:11,902 - INFO - Initial cleanup completed - all previous files deleted
2025-07-25 08:29:18,193 - INFO - File encrypted successfully in 0.03 seconds
2025-07-25 08:29:18,194 - INFO - File encrypted and saved in 0.03 seconds
2025-07-25 08:29:19,270 - INFO - File securely stored: template_surat_rekomendasi_rbb2025.docx → 07e281b4-c779-4f54-a737-8f7a2da8cd91.docx
2025-07-25 08:36:15,066 - INFO - User 923869729 (grndsard) started the bot
2025-07-25 08:36:15,446 - INFO - Welcome message sent to user 923869729 in en language
2025-07-25 08:36:34,107 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 08:36:34,114 - INFO - File encrypted and saved in 0.01 seconds
2025-07-25 08:36:35,242 - INFO - File securely stored: KESELAMATAN DAN KESEHATAN KERJA DI LABORATORIUM PRINT.docx → 6b21086c-ec1d-426e-9209-3aa0f2aa8a51.docx
2025-07-25 08:50:48,730 - INFO - Deleted old file: files/07e281b4-c779-4f54-a737-8f7a2da8cd91.docx
2025-07-25 08:53:48,755 - INFO - Deleted old file: files/6b21086c-ec1d-426e-9209-3aa0f2aa8a51.docx
2025-07-25 08:58:58,640 - INFO - User 923869729 (grndsard) started the bot
2025-07-25 08:58:59,474 - INFO - Welcome message sent to user 923869729 in en language
2025-07-25 08:59:17,826 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 08:59:17,831 - INFO - File encrypted and saved in 0.01 seconds
2025-07-25 08:59:18,909 - INFO - File securely stored: JUSTIFIKASI_047_V_2023_Pengadaan_Kabel_Charger_Yusitania_dan_RIdwan.docx → a42a70ae-396c-41e2-9fc1-55147b086171.docx
2025-07-25 09:08:47,506 - INFO - User 923869729 (grndsard) clicked: start_bot
2025-07-25 09:14:00,296 - INFO - Secure RupaGanti Bot starting with enhanced AES-256 with hardware acceleration encryption...
2025-07-25 09:14:00,316 - INFO - Database cleaned on startup
2025-07-25 09:14:00,317 - INFO - Startup cleanup: Deleted files\a42a70ae-396c-41e2-9fc1-55147b086171.docx
2025-07-25 09:14:00,318 - INFO - Initial cleanup completed - all previous files deleted
2025-07-25 09:14:11,134 - INFO - User 923869729 (grndsard) started the bot
2025-07-25 09:14:12,021 - INFO - Welcome message sent to user 923869729 in en language
2025-07-25 09:14:26,877 - INFO - File encrypted successfully in 0.01 seconds
2025-07-25 09:14:26,881 - INFO - File encrypted and saved in 0.01 seconds
2025-07-25 09:14:28,009 - INFO - File securely stored: JUSTIFIKASI_047_V_2023_Pengadaan_Kabel_Charger_Yusitania_dan_RIdwan.docx → 174fb130-19bb-4ab0-9609-41a345f80fee.docx
2025-07-25 09:14:41,298 - INFO - User 923869729 (grndsard) clicked: 9_1
2025-07-25 09:14:42,055 - INFO - File decrypted successfully with AES in 0.01 seconds
2025-07-25 09:14:48,859 - WARNING - docx2pdf failed: Word.Application.Quit
2025-07-25 09:14:51,684 - INFO - Cleaned up failed file: files/174fb130-19bb-4ab0-9609-41a345f80fee.docx
2025-07-25 09:15:12,522 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-25 09:15:14,680 - INFO - User 923869729 (grndsard) started the bot
2025-07-25 09:15:15,633 - INFO - Welcome message sent to user 923869729 in en language
2025-07-25 09:15:26,477 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 09:15:26,483 - INFO - File encrypted and saved in 0.01 seconds
2025-07-25 09:15:27,711 - INFO - File securely stored: Format Uraian Pekerjaan.xlsx → f5dcc92b-abda-4928-ac01-b2549de2195c.xlsx
2025-07-25 09:15:32,959 - INFO - User 923869729 (grndsard) clicked: 9_2
2025-07-25 09:15:33,371 - ERROR - Error updating countdown: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: message to edit not found
2025-07-25 09:15:33,374 - ERROR - Error in check_session: 923869729
2025-07-25 09:15:33,720 - INFO - File decrypted successfully with AES in 0.00 seconds
2025-07-25 09:15:35,951 - WARNING - docx2pdf failed: (-2147352567, 'Exception occurred.', (0, 'Microsoft Word', 'The file appears to be corrupted.', 'wdmain11.chm', 25272, -2146822496), None)
2025-07-25 09:15:35,964 - ERROR - Word to PDF error: file 'C:\Users\Grandis\AppData\Local\Temp\rupaganti_un1k_i5p\input_2.docx' is not a Word file, content type is 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml'
2025-07-25 09:15:35,965 - INFO - Cleaned up failed file: files/f5dcc92b-abda-4928-ac01-b2549de2195c.xlsx
2025-07-25 09:15:51,557 - INFO - User 923869729 (grndsard) clicked: no_thanks
2025-07-25 09:15:54,313 - INFO - User 923869729 (grndsard) started the bot
2025-07-25 09:15:54,716 - INFO - Welcome message sent to user 923869729 in en language
2025-07-25 09:16:05,942 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 09:16:05,946 - INFO - File encrypted and saved in 0.01 seconds
2025-07-25 09:16:07,040 - INFO - File securely stored: 1. Berita Acara Serah Terima Pekerjaan.doc → d8b7b511-dc7c-4113-ae9d-efca01aa2c3e.doc
2025-07-25 09:18:10,093 - INFO - Deleted expired file: files/d8b7b511-dc7c-4113-ae9d-efca01aa2c3e.doc
2025-07-25 09:22:38,026 - INFO - Secure RupaGanti Bot starting with enhanced AES-256 with hardware acceleration encryption...
2025-07-25 09:22:38,031 - INFO - Database cleaned on startup
2025-07-25 09:22:38,031 - INFO - Initial cleanup completed - all previous files deleted
2025-07-25 09:22:46,557 - INFO - User 923869729 (grndsard) started the bot
2025-07-25 09:22:47,375 - INFO - Welcome message sent to user 923869729 in en language
2025-07-25 09:23:20,728 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 09:23:20,730 - INFO - File encrypted and saved in 0.00 seconds
2025-07-25 09:23:21,760 - INFO - File securely stored: WhatsApp Image 2024-07-19 at 13.54.04.jpeg → 3b39b801-4486-428a-b223-5a4d5c8b771c.jpeg
2025-07-25 09:23:23,495 - INFO - Cleaned up failed file: files/3b39b801-4486-428a-b223-5a4d5c8b771c.jpeg
2025-07-25 09:27:47,833 - INFO - Secure RupaGanti Bot starting with enhanced AES-256 with hardware acceleration encryption...
2025-07-25 09:27:47,837 - INFO - Database cleaned on startup
2025-07-25 09:27:47,838 - INFO - Initial cleanup completed - all previous files deleted
2025-07-25 09:27:51,256 - INFO - User 923869729 (grndsard) clicked: start_bot
2025-07-25 09:28:16,808 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 09:28:16,811 - INFO - File encrypted and saved in 0.01 seconds
2025-07-25 09:28:17,877 - INFO - File securely stored: image_2024-10-28_10-23-09.png → e3c9d572-56cc-43ab-8ec0-efda773140a0.png
2025-07-25 09:28:34,078 - INFO - User 923869729 (grndsard) clicked: 1_1
2025-07-25 09:28:35,180 - INFO - File decrypted successfully with AES in 0.00 seconds
2025-07-25 09:28:36,912 - INFO - Original file deleted after processing: files/e3c9d572-56cc-43ab-8ec0-efda773140a0.png
2025-07-25 09:28:45,172 - INFO - User 923869729 (grndsard) clicked: yes_more
2025-07-25 09:28:55,956 - INFO - File encrypted successfully in 0.00 seconds
2025-07-25 09:28:55,958 - INFO - File encrypted and saved in 0.00 seconds
2025-07-25 09:28:56,998 - INFO - File securely stored: NB PRogress.xlsx → 4830429e-d381-43c6-92d2-5522a7827417.xlsx
2025-07-25 09:29:04,740 - INFO - User 923869729 (grndsard) clicked: compress_zip_2
2025-07-25 09:29:05,575 - INFO - File decrypted successfully with AES in 0.01 seconds
2025-07-25 09:29:07,054 - INFO - Original file deleted after processing: files/4830429e-d381-43c6-92d2-5522a7827417.xlsx
2025-07-25 09:29:10,654 - INFO - User 923869729 (grndsard) clicked: yes_more
2025-07-25 09:34:32,572 - INFO - User 923869729 (grndsard) clicked: cancel_2
2025-07-25 09:34:32,930 - ERROR - Failed to read or decrypt file files/4830429e-d381-43c6-92d2-5522a7827417.xlsx: [Errno 2] No such file or directory: 'files/4830429e-d381-43c6-92d2-5522a7827417.xlsx'
2025-07-25 09:34:39,763 - INFO - User 923869729 (grndsard) clicked: start_bot
